.cirrus.yml

```diff
@@ -243,5 +248,6 @@ static_binary_task:
         mkdir -p /nix
         podman run --rm --privileged -ti -v /:/mnt nixos/nix cp -rfT /nix /mnt/nix
         podman run --rm --privileged -ti -v /nix:/nix -v ${PWD}:${PWD} -w ${PWD} nixos/nix nix --print-build-logs --option cores 8 --option max-jobs 8 build --file nix/
```

Comment:
Do you think we can try to stick to nixos/nix directly without wrapping a virtual machine?

Reply:
I tried that multiple times before, but the key difficulty is: `nixos/nix` DON'T have bash.

If we are running with CI + docker directly, usually it assumed with bash exists, which make the test failed :-(

Reply:
P.S. I also try to create another nix-based container with Ubuntu 20.04 + bash (https://hub.docker.com/r/alvistack/nix) but this image generate another crazy behavior when integrating with CI :-(